app/controllers/courses/courses_controller.rb

```diff
@@ -0,0 +1,39 @@
+class Courses::CoursesController < Courses::Controllers
+  load_and_authorize_resource
+
+  def show
```

Comment:
Might want to annotate these methods as `:nodoc:` since controller actions are self-explanatory.